src/static/css/module/loan-comparison.less

```diff
@@ -1052,3 +1048,26 @@ table.unstyled {
   display: none;
 }
 
+/* Title Heading */
+.round-step {
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 1.5em;
+  line-height: 1.5;
+  text-align: center;
+  background-color: @green-tint;
```

Comment:
I think we could use `.icon-round` as a mixin here instead of these first six lines...